sdk/communication/communication-phone-numbers/samples-dev/getPurchasedPhoneNumber.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to us the PhoneNumbersClient to get a purchased phone number.
```

Comment:
nit: Demonstrates how to *use* the PhoneNumbersClient to get a purchased phone number.